README.md

```diff
@@ -117,59 +117,124 @@ Markers geometry writes as posNode, for all other layers geometry writes as posL
 
 # Filter
 
-OGC Filter realization:
-
-## GmlObjectID filter
+Realization of OGC Filter Encoding v1.1.0
+
+Filter implementations return only inner content of filter element.
+
+Some considerations for all filter constructors:
+* "expression" - propertyName, literal, operator filter or function
+* "propertyExpression" - if argument on this position is not Element and is not "expression" method it suspect to be a propertyName
+* "literalExpression" - if argument on this position is not Element and is not "expression" it suspect to be a literal
+
+|Name|Constructor|
```

Comment:
In addition to the name and constructor description is needed too. Also each filter options must be described somehow.